src/EA.Weee.Web/Areas/Scheme/Views/Shared/_ReviewSubmittedEvidencePartial.cshtml

```diff
@@ -94,4 +94,12 @@
             }
         </tbody>
     </table>
+    <br />
+    @{
+        Html.RenderPartial("~/Views/Shared/_EvidenceNotesPagerPartial.cshtml", Model.EvidenceNotesDataList, new ViewDataDictionary()
+                    { { "pagerRouteValues", new { tab = Model.ActiveDisplayOption.ToDisplayString(),
+                         manageEvidenceNoteViewModel_SelectedComplianceYear = Model.ManageEvidenceNoteViewModel.SelectedComplianceYear,
+                         manageEvidenceNoteViewModel_RecipientWasteStatusFilterViewModel_NoteStatusValue =
```

Comment:
@GuidoWorkDev  We dont need these route values on the paging as there are no filters on this screen